src/makeRobotAccountant.js

```diff
@@ -6,7 +6,19 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
-}
+  let call = 0;
+  const ErrorMessage = `Bzzz... Error!`;
+
+  return (firstNumber) => {
+    call++;
```

Comment:
I think it will be better to do it in next returned function.
I dont know why))) 
it just looks better, when first functions just returns second, and all operations are located in last)

however, it is not fundamental))